openfl-workspace/torch_llm_horovod/src/InHorovodrun.py

```diff
@@ -9,7 +9,7 @@
 
 import horovod.torch as hvd
 
-import openfl.native as fx
```

Comment:
Why is this change of `InHorovodrun.py` needed, given that at this stage we're just deprecating the Python API?